src/Data/Nat/Binary/Minus.agda

```diff
@@ -0,0 +1,239 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Subtraction on Bin and some of its property proofs.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Nat.Binary.Minus where
+
+open import Data.Nat.Binary.Base
+open import Data.Nat.Binary.Properties
+open import Data.Empty using (⊥)
+open import Data.Nat as ℕ using (ℕ)
+import Data.Nat.Properties as ℕₚ
+open import Data.Product using (_×_; _,_; proj₁; proj₂; ∃)
+open import Data.Sum using (inj₁; inj₂)
+open import Function using (_∘_; _$_)
+open import Relation.Binary using (Tri; _Preserves_⟶_; _Preserves₂_⟶_⟶_)
+open import Relation.Binary.PropositionalEquality as P using
+  (_≡_; _≢_; _≗_; refl; sym; trans; cong; cong₂; resp₂; subst)
+import Relation.Binary.Reasoning.Base.Triple as InequalityReasoning
+open import Relation.Nullary using (Dec; yes; no)
+open import Relation.Nullary.Negation using (contradiction)
+open import Algebra.FunctionProperties {A = ℕᵇ} _≡_
+open import Algebra.Properties.CommutativeSemigroup +-commSemigroup
+     using (xy∙z≈y∙xz; x∙yz≈y∙xz; xy∙zu≈xz∙yu)
+
+private
+  module IneqReasoning = InequalityReasoning {A = ℕᵇ} {_≈_ = _≡_} ≤-isPreorder
+                                   <-trans (resp₂ _<_) <⇒≤ <-≤-trans ≤-<-trans
+open ℕᵇ
+open Tri
+open _<_
```

Comment:
Don't need any of these 3 `open` statements

Reply:
``open Tri`` is used in new Minus.agda, and I remove the other two.

Reply:
You don't need `open Tri` either.

Reply:
You seem to refer to old Minus.agda while there is suggested the commit of December 23
(which uses the constructors of Tri).

Reply:
Your commit on December 23 6b3c586 still opens `Tri` but as far as I can see the file no longer uses it as we've switched to a different definition?

Reply:
The new Minus.agda
* has a simpler implementation for ``∸`` (as you suggested),
* has about twice as much functionality (the first version was not complete, it was to see the reaction and then to decide),
* uses the constructors of Tri, for example, in ``x<y⇒0<y∸x``.

Following your suggestion, I am going to save the fresh comments on GitHub, delete this my branch, and to start a new PR with Minus.agda and without those minor functions mentioned.